shared/src/main/scala/amf/core/client/scala/resource/ResourceLoader.scala

```diff
@@ -6,9 +6,10 @@ import scala.concurrent.Future
 
 trait ResourceLoader {
 
-  /** Fetch specified resource and return associated content. Resource should have benn previously accepted. */
+  /** Fetch specified resource and return associated content. Resource should have been previously accepted.
+    * If the resource not exists, you should return a future failed with an ResourceNotFound exception. */
```

Comment:
`doesn't exist, it returns a failed future due to a ResourceNotFound exception` or something like that.

Reply:
*caused by a Resour.... could also be appropiate.